lib/attrsets.nix

```
@@ -1209,6 +1246,41 @@ rec {
     in
     recurse [ ] set;
 
+  /**
+    Like [`lib.attrsets.mapAttrsRecursiveCond`](#function-library-lib.attrsets.mapAttrsRecursiveCond) but also recurses on lists. In other words it recurses over an arbitrary nesting of attrsets and lists.
+
+    :::{#map-data-recursive-cond-example .example}
```

```suggestion
    # Examples
    :::{#map-data-recursive-cond-example .example}
```

See https://github.com/NixOS/nixpkgs/tree/master/doc#documentation-conventions (scroll down to "When documenting functions...")

Also please match the conventional Inputs, Type, Examples structure